docs/reference/ilm/example-index-lifecycle-policy.asciidoc

```diff
@@ -36,130 +41,78 @@ log data has the following data retention guidelines:
+* A host with the {agent} installed and configured to send logs to your {es}
+cluster.
 
 [discrete]
-[[example-using-index-lifecycle-policy-view-fb-ilm-policy]]
-==== View the {filebeat} {ilm-init} policy
+[[example-using-index-lifecycle-policy-view-ilm-policy]]
+==== View the policy
 
-{filebeat} includes a default {ilm-init} policy that enables rollover. {ilm-init}
-is enabled automatically if you’re using the default `filebeat.yml` and index template.
+The {agent} uses data streams with an index pattern of `logs-*-*` to store log
+monitoring data. {es}'s built-in `logs` {ilm-init} policy automatically manages
+backing indices for these data streams.
 
-To view the default policy in {kib}:
+To view the `logs` policy in {kib}:
 
-. Go to Management and select *Index Lifecycle Policies*.
-. Search for _filebeat_
-. Select the _filebeat-version_ policy.
+. Open the menu and go to **Stack Management > Index Lifecycle Policies**.
+. Select the `logs` policy.
 
-This policy initiates the rollover action when the index size reaches 50GB or
-becomes 30 days old.
+The `logs` policy uses our recommended rollover defaults. These defaults
+initiate a rollover when the current write index reaches 50GB or becomes 30 days
+old.
 
-[role="screenshot"]
-image::images/ilm/tutorial-ilm-hotphaserollover-default.png["Default policy"]
+To confirm these rollover settings, click **Settings** for the hot phase. Then
+disable **Use recommended defaults** to display the rollover settings.
 
+[role="screenshot"]
+image::images/ilm/tutorial-ilm-hotphaserollover-default.png[View rollover defaults]
 
 [discrete]
+[[ilm-ex-modify-policy]]
 ==== Modify the policy
 
-The default policy is enough to prevent the creation of many tiny daily indices.
-You can modify the policy to meet more complex requirements.
+The `logs` policy's default settings are enough to prevent the creation of many
+tiny daily indices. You can modify the policy to meet more complex requirements.
 
-. Activate the warm phase.
+. Activate the warm phase and click **Settings**.
 +
 --
 [role="screenshot"]
-image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png["Modify to add warm phase"]
-
-.. Set one of the following options to control when the index moves to the warm phase:
-
-*** Provide a value for *Timing for warm phase*. Setting this to *15* keeps the
-indices on hot nodes for a range of  15-45 days, depending on when the initial
-rollover occurred.
+image::images/ilm/tutorial-ilm-modify-default-warm-phase-rollover.png[Add a warm phase with custom settings]
 
-*** Enable *Move to warm phase on rollover*. The index might move to the warm phase
-more quickly than intended if it reaches the *Maximum index size* before the
-the *Maximum age*.
+.. Set *Move data into phase when* to *30 days*. This keeps indices in the hot
+tier for 30 days. After 30 days, indices move to the warm tier.
```

Comment:
As mentioned above, this keeps the data in the hot phase for 30 days after rollover.
```suggestion
.. Set *Move data into phase when* to *30 days*. This moves indices to the warm tier 30 days after rollover.
```